samples/Microsoft.AspNet.SignalR.ClientVB.Samples/Client.vb

```diff
@@ -0,0 +1,49 @@
+﻿Imports System
+Imports System.IO
+Imports System.Text
+Imports Microsoft.AspNet.SignalR.Client
+Imports Microsoft.AspNet.SignalR.Hubs
+
+Public Class Client
+    Private _traceWriter As TextWriter
+    Public Sub New(writer As TextWriter)
+        _traceWriter = writer
+    End Sub
+    Public Sub Run(url As String)
+        Try
+            Dim HubConnection = New HubConnection(url)
+            HubConnection.TraceWriter = _traceWriter
+            HubConnection.TraceLevel = TraceLevels.Events
+
+            Dim hubProxy = HubConnection.CreateHubProxy("chat")
+            'Register client function and behavior
+            hubProxy.On(Of String)("send", Sub(data) HubConnection.TraceWriter.WriteLine(data))
+
+            HubConnection.Start().Wait()
```

Comment:
Use async/await instead of `.Wait()` - while in this case it does not matter much because it is a Console.App this is a sample and people copy the code and then deadlock themselves.